jobs/JGLOBAL_ATMOS_UPP

```diff
@@ -0,0 +1,48 @@
+#! /usr/bin/env bash
+
+source "${HOMEgfs}/ush/preamble.sh"
+source "${HOMEgfs}/ush/jjob_header.sh" -e "post" -c "base post"
+
+##############################################
+# Set variables used in the exglobal script
+##############################################
+
+##############################################
+# Begin JOB SPECIFIC work
+##############################################
+
+# Construct COM variables from templates
+YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_RESTART COM_ATMOS_ANALYSIS COM_ATMOS_HISTORY COM_ATMOS_MASTER
+if [[ ! -d ${COM_ATMOS_MASTER} ]]; then mkdir -m 775 -p "${COM_ATMOS_MASTER}"; fi
+
+if [[ ${GOESF} == "YES" ]]; then
+  YMD=${PDY} HH=${cyc} generate_com -rx COM_ATMOS_GOES
+  if [[ ! -d ${COM_ATMOS_GOES} ]]; then mkdir -m 775 -p "${COM_ATMOS_GOES}"; fi
+fi
```

Comment:
I will likely remove this `COM_ATMOS_GOES` from this script as the master file should go back to `COM_ATMOS_MASTER`.

`COM_ATMOS_GOES` can contain the interpolated grib2 products from the master file.